fbchat/graphql.py

```diff
@@ -44,7 +44,7 @@ def get_customization_info(thread):
         rtn['nicknames'] = {}
         for k in info['participant_customizations']:
             rtn['nicknames'][k['participant_id']] = k.get('nickname')
-    else:
+    elif info['participant_customizations']:
```

Comment:
You are right in this change! But if `info['participant_customizations']` is not set, this will still throw an error. Maybe we should have it be `info.get('participant_customizations')`?